src/main/java/com/bento909/CsvParser.java

```diff
@@ -29,9 +32,7 @@ public CsvParser(String filePath) {
 
     private List<String> parseNextLine() throws IOException {
         if (scanner.hasNextLine()) {
-            String line = scanner.nextLine();
-            String[] tokens = line.split(",");
-            return Arrays.asList(tokens);
+            return SPLITTER.splitToList(scanner.nextLine());
```

Comment:
Guava has the concept of an immutable list. Use should use it here and also return an empty immutable list below if no line is found in the if condition above.

https://github.com/google/guava/wiki/ImmutableCollectionsExplained